packages/integrations/cloudflare/src/index.ts

```diff
@@ -111,13 +141,46 @@ export default function createIntegration(args?: Options): AstroIntegration {
 				addWatchFile,
 				addMiddleware,
 			}) => {
+				let session = config.session;
+
+				const isBuild = command === 'build';
+
+				if (config.experimental.session && !session?.driver) {
+					const bindingName = args?.sessionKVBindingName ?? 'SESSION';
+					logger.info(
+						`Configuring experimental session support using ${isBuild ? 'Cloudflare KV' : 'filesystem storage'}`,
+					);
+					logger.info(`Be sure to define a KV binding named "${bindingName}".`);
+					session = isBuild
+						? {
+								...session,
+								driver: 'cloudflare-kv-binding',
+								options: {
+									// This is the name of the global variable where the KV namespace is stored
+									// in the worker. It is assigned to this inside the request handler.
+									binding: '__ASTRO_SESSION',
```

Comment:
Whether this is a nitpick or not, I foresee a use case where users might want to override this because they don't want to leak information relative to their framework. However, I think it's fine to keep it like this for now, and re-iterate if users need to overrde it

Reply:
They can override the actual binding name: this is just the name of the variable we use to pass it to unstorage.

Reply:
I've changed it to use the same name as the binding, because otherwise the error message from unstorage would be confusing.